### PR TITLE
External Bank Accounts Component - Fix/Refactor

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [X] Bug fix

### Linked issue

- https://cybrid.atlassian.net/browse/CYB-1111
- https://cybrid.atlassian.net/browse/CYB-983

### Why

Added:

- List of External Bank accounts connected
- Detail of each account
- Modal to disconnect account
- Hide add account button
- More tests
- Update dependencies

### Evidence

https://user-images.githubusercontent.com/7268597/220967819-52e955f2-6b95-49b4-a370-2cdc5bc80c6a.mov

### CODE REVIEW

File: CybridSDKTestApp/CryptoAuthenticator.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ import Foundation
 class CryptoAuthenticator {
 
     private let session: URLSession
-    private let params = "banks:read banks:write accounts:read accounts:execute customers:read customers:write customers:execute prices:read quotes:execute trades:execute trades:read"
+    private let params = "banks:read banks:write accounts:read accounts:execute customers:read customers:write customers:execute prices:read quotes:execute trades:execute trades:read workflows:execute workflows:read external_bank_accounts:execute external_bank_accounts:read transfers:read transfers:execute"
```
Comment (brent-cybrid) [inline CybridSDKTestApp/CryptoAuthenticator.swift:14]: Should be:

```
banks:read
banks:write                                      
accounts:read                                    
accounts:execute                                 
customers:read                                   
customers:write                                  
customers:execute                                
prices:read                                      
quotes:execute                                   
quotes:read                                      
trades:execute                                   
trades:read                                      
transfers:execute                                
transfers:read                                   
rewards:execute
rewards:read
external_bank_accounts:read
external_bank_accounts:write
external_bank_accounts:execute
workflows:read
workflows:execute
deposit_addresses:read
deposit_addresses:execute
```

File: CybridSDKTestApp/Info.plist [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
-	<key>NSPhotoLibraryUsageDescription</key>
-	<string>Photos permission to Identity Verification</string>
 	<key>CybridClientId</key>
 	<string></string>
 	<key>CybridClientSecret</key>
 	<string></string>
 	<key>CybridCustomerGUID</key>
 	<string></string>
+	<key>NSCameraUsageDescription</key>
+	<string>Camera permission to Identity Verification</string>
```
Comment (brent-cybrid) [inline CybridSDKTestApp/Info.plist:12]: `for` instead of `to`

File: CybridSDKTests/Components/BankAccounts/View/BankAccountsViewModelTest.swift [hunk 3]
```diff
@@ -30,6 +31,23 @@ class BankAccountsViewModelTest: XCTestCase {
         XCTAssertNil(viewModel.workflowJob)
     }
 
+    func test_fetchExternalBankAccounts_Successfully() {
```
Comment (brent-cybrid) [inline CybridSDKTests/Components/BankAccounts/View/BankAccountsViewModelTest.swift:34]: We have negative testing for this and the disconnect?

**Review by brent-cybrid — APPROVED** [review]

File: CybridSDKTestApp/Info.plist (hunk 2 above)
Comment (nalancer08) [reply CybridSDKTestApp/Info.plist:12]: Done

File: CybridSDKTestApp/CryptoAuthenticator.swift (hunk 1 above)
Comment (nalancer08) [reply CybridSDKTestApp/CryptoAuthenticator.swift:14]: Done

**Pull request merged**